src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -90,7 +102,7 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
       isOwnSignedPost={isOwnSignedPost}
       message={item?.description}
-      name={postMaker?.data?.username}
+      name={anonUsername}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `postMaker?.data?.username` with `anonUsername` is a good change. It ensures that if the post maker is an anonymous user, their official anonymous username is displayed, maintaining consistency across the application.
```diff
- name={postMaker?.data?.username}
+ name={anonUsername}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->